DECISION_LOG.md

```diff
@@ -7,6 +7,6 @@
  
 The original idea was to explicitly support retry in the scheduler. However, this turned out to complicate the algorithm
 in case of event sources. Mostly it would be harder to manage the buffer, and the other event sources, thus what
-does it mean for other event sources that there was a failed controller execution? Probably it would be better to
-manage this in an abstract controller, and just use the "reprocess event-source" source in case of an error. 
+does it mean for other event sources that there was a failed controllerConfiguration execution? Probably it would be better to
```

Comment:
Looks like you got too eager with search and replace :)

Reply:
It's intellij too intelligent :) will fix these

Reply:
but probably we can delete Decision log since it is not used